### DISCUSSION TITLE
第1节课笔记(14班)

### DISCUSSION THREAD

**Discussion by NewDaNew** [InternLM/Tutorial#61] [笔记]
14班大模型培训笔记第一节课

**Comment by NewDaNew** [comment]
大家记得在下方提交笔记哦，可以是知乎，CSDN等平台的学习笔记。好的学习笔记可以帮助大家巩固知识

**Reply by wakefuldreamer** [reply]
https://www.yuque.com/gggggenius/fmhsbr/zk3nyzww9x8nsrh2?singleDoc# 《第1课》

**Comment by ZZfive** [comment]
https://sincere-papyrus-971.notion.site/2eb29a179c454485b63e88be77b42458?pvs=4

**Comment by Mercuryoo** [comment]
https://t3d0h0gdxm.feishu.cn/docx/CX6odkxwUoT4ThxnfrfcI7fTn7w?from=from_copylink

**Comment by Xinyusei** [comment]
[第一次课笔记](https://github.com/Xinyusei/InternLM/blob/main/Notes1/Notes.md)

**Comment by zenith779** [comment]
https://www.wolai.com/jinagzk2018/5UaqA1jmozTVj2BcApAG6Q

**Comment by tianyabanbu** [comment]
https://qiz21j4rsiq.feishu.cn/docx/A0JAdkPqPoexPhx3miNcJKmSn1e

**Comment by Nobitaxi** [comment]
https://blog.csdn.net/weixin_44288092/article/details/135388145?spm=1001.2014.3001.5501

**Comment by 0Pinky0** [comment]
https://www.yuque.com/0pinky0/acxzx1/ams4a01lty4xax0o?singleDoc# 《第一节 书生·浦语大模型全链路开源开放体系》

**Comment by Seanking756** [comment]
https://blog.csdn.net/Sean7566/article/details/135412049

**Comment by SingaCycle** [comment]
https://blog.csdn.net/z1370378301/article/details/135418130

**Comment by shiningwhite-cmd** [comment]
https://www.yuque.com/shanxiaobai/xt4xo5/glwb7qatigrve3pc?singleDoc# 《InternML》

**Comment by shiningwhite-cmd** [comment]
https://www.yuque.com/shanxiaobai/xt4xo5/glwb7qatigrve3pc?singleDoc# 《InternML》

**Comment by BanTomato** [comment]
https://github.com/BanTomato/internLMNotes/blob/main/%E5%A4%A7%E6%A8%A1%E5%9E%8BOpenMMLab.md

**Comment by lizihe11** [comment]
https://gitee.com/kiddylzh/intern-lm-tutorial-note/blob/master/note/first%20note.md

**Comment by xilimala** [comment]
https://leaf-fennel-588.notion.site/9d79236431f3429680c8206e52e0d34e?pvs=4
